schemas/app-sre/app-1.yml

```diff
@@ -324,3 +329,9 @@ required:
 - labels
 - name
 - description
+- escalationPolicy
```

Comment:
Same here.

Remove all the changes of adding required field for an existing schema file.